packages/wp-story-editor/src/components/header/buttons/index.js

```diff
@@ -96,6 +98,10 @@ function Buttons() {
   return (
     <ButtonList>
       <List>
+        <UndoButton />
+        <Space />
```

Comment:
Could we redo this with `gap` in CSS instead? It wasn't well supported originally, but is now and is a prettier solution than spacer elements (and fully accessible).